test/numerical/TestGRU.cpp

```diff
@@ -78,22 +104,32 @@ int main(int argc, char *argv[]) {
     for (int64_t b = 3; b < 4; b++)
       for (int64_t i = 2; i < 5; i++)
         for (int64_t h = 2; h < 5; h++)
+        // clang-format off
+#ifdef TEST_GRU_L1
+          {
+            int64_t l = 1;
+#else
           for (int64_t l = 0; l < 2; l++) {
+#endif
```

Comment:
Please keep the loop in all cases, but define this before
```
#ifdef TEST_GRU_1
  int l_min = 1
#else 
  int l_min = 0
#endif 
```
and then for (l=l_min, l<2l ++l)
```

Reply:
Done. Thanks!